hetio/neo4j.py

```diff
@@ -8,14 +8,28 @@
 import py2neo.packages.httpstream
 import pandas
 
+from tqdm import tqdm
+from functools import reduce
+from operator import or_
+
+PY2NEO_VER = int(py2neo.__version__[0])
+
 import hetio.hetnet
+import hetio.stats
 
 # Avoid SocketError
 py2neo.packages.httpstream.http.socket_timeout = 1e8
 
-def export_neo4j(graph, uri, node_queue=100, edge_queue=100):
+def export_neo4j(graph, uri, node_queue=100, edge_queue=100, show_progress=False):
     """Export hetnet to neo4j"""
-    db_graph = py2neo.Graph(uri)
+
+    if PY2NEO_VER == 2:
```

Comment:
Can we change the if statement to evaluate `isinstance(uri, py2neo.Graph)` rather than to evaluate the version? This way for py2neo 2 you could pass a `py2neo.Graph`. And I'm hopeful that https://github.com/technige/py2neo/issues/605 will eventually be fixed.